packages/clay-css/src/images/icons/flags-en-CA.svg

```diff
@@ -0,0 +1,13 @@
+<!--
+* SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2022 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
+
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg">
+    <path d="M16 2H0v12h16V2Z" fill="#B0BEC9"/>
```

Comment:
We should use tabs instead of spaces for indenting.

Reply:
@ilzamcmed we still need to do this. Change the indent to tab instead of space.

![spaces-to-tabs](https://github.com/liferay/clay/assets/788266/d3ff9682-7999-4d73-90dc-c434091692f9)

Reply:
Not sure why it's adding space over here, when I press tab, somehow it's recognized by space. I'll see what's going on and send the correct format.
Thanks @pat270

Reply:
Fixed! My indentation was broke :disappointed: